src/App/Settlements/helpers.ts

```diff
@@ -1,5 +1,248 @@
+import { strict as assert } from 'assert';
 import moment from 'moment';
-import { DateRanges, SettlementStatus, SettlementFilters } from './types';
+import ExcelJS from 'exceljs';
```

Comment:
All this work because it couldn't be in JSON :cry:

Reply:
:sob: yeah but I get it, bank back office staff don't deal in JSON, they deal in Excel. I guess if someone was processing this automatically.. well we'd do something very different in that case anyway.